javaslang/generator/Generator.scala

```diff
@@ -2812,6 +2836,7 @@ def generateTestClasses(): Unit = {
 
         val test = im.getType("org.junit.Test")
         val assertThat = im.getStatic("org.assertj.core.api.Assertions.assertThat")
+        val assertThatThrownBy = im.getStatic("org.assertj.core.api.Assertions.assertThatThrownBy")
```

Comment:
could you move this into `${(i == 1 && !checked)` branch to avoid a lot of unused imports?